demos.md

```diff
@@ -0,0 +1,18 @@
+# Demos
+A demo is usually presented at the end of each sprint,
```

Comment:
I'd say "each *development* sprint". For me, a simple "sprint" is about short-distance running,

Reply:
I am not sure what part you suggest to change here.
Do you mean?: "At the end of each development sprint a demo is presented descr..."

Reply:
Correct.